src/lib/utils/series-utils.js

```diff
@@ -80,14 +91,17 @@ export function getStackedData(children, attr) {
     }
     const attr0 = `${attr}0`;
     childData.push(data.map((d, dIndex) => {
-      let prevValue = 0;
-      if (prevIndex >= 0) {
-        const prevD = childData[prevIndex][dIndex];
-        prevValue = prevD[attr0] + prevD[attr];
+      // In case if it's the first series don't try to override any values.
+      if (prevIndex < 0) {
```

Comment:
Might want to consider checking this before the line 93 (`childData.push`) so you can push without looping doing the `.map`.